docs/content/engines/hive.md

```diff
@@ -228,6 +228,18 @@ OK
 
 ```
 
+## Setting of Location
+In addition to the way setting location above, you can also place the location setting in TBProperties
+to avoid Hive accessing Paimon's location through its own file system when creating tables.
+This method is effective in scenarios using Object storage,such as s3.
+
+```sql
+CREATE EXTERNAL TABLE external_test_table
```

Comment:
You can just document this in `Hive SQL: access Paimon Tables by External Table`.